web/styles/zulip.css

```diff
@@ -2093,3 +2130,11 @@ body:not(.hide-left-sidebar) {
         border: 1px solid var(--color-border-personal-menu-avatar);
     }
 }
+
+.typeahead.dropdown-menu .active a .zulip-icon-bot {
+    color: var(--color-icon-bot);
+}
+
+.mention-content-wrapper .zulip-icon-bot {
+    margin-top: 100px;
+}
```

Comment:
Where does this 100px magic number come from?

Reply:
Sorry about this one, I was just testing few things and by mistake i committed this code .  I am really sorry